validation/biogeochemistry/settling_diffusion.jl

```diff
@@ -0,0 +1,74 @@
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.BoundaryConditions: fill_halo_regions!, ImpenetrableBoundaryCondition
+using Printf
+using GLMakie
+
+# Buoyancy profile
+bᵢ(x, y, z) = 1e-5 * z
+b_particle = bᵢ(0, 0, 0) # particle buoyancy
+ν = 1e-4
+κ_particle = 1e-3
+R_particle = 1.0
+Δt = 1e-2
+
+grid = RectilinearGrid(size=128, z=(-4, 4), topology=(Flat, Flat, Bounded))
+
+no_penetration = ImpenetrableBoundaryCondition()
+slip_bcs = FieldBoundaryConditions(grid, (Center, Center, Face),
+                                   top=no_penetration, bottom=no_penetration)
```

Comment:
Awesome! Want to use this for the NP model.

Reply:
nice